src/Api/Billing/Controllers/TaxController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Bit.Api.Billing.Models.Requests;
+using Bit.Api.Billing.Models.Responses;
+using Bit.Core.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.Billing.Controllers;
+
+[Route("tax")]
+[Authorize("Application")]
+public class TaxController(
+    IStripeAdapter stripeAdapter) : Controller
+{
+    [HttpPost]
+    [Route("calculate")]
+    public async Task<IResult> CalculateAsync([FromBody] CalculateTaxRequestModel requestBody)
+    {
+        var options = new Stripe.Tax.CalculationCreateOptions
+        {
+            Currency = "usd",
+            CustomerDetails = new()
+            {
+                Address = new()
+                {
+                    PostalCode = requestBody.BillingAddressPostalCode,
+                    Country = requestBody.BillingAddressCountry
+                },
+                AddressSource = "billing"
+            },
+            LineItems = new()
+            {
+                new()
```

Comment:
If it gets more complex we need to calculate this with the product identifier from Stripe if we use different tax codes per product.